db/views/blood_pressures_per_facility_per_days_v02.sql

```diff
@@ -0,0 +1,19 @@
+WITH latest_bp_per_patient_per_day AS
+ (SELECT DISTINCT ON (facility_id, blood_pressures.patient_id, year, day)
+      blood_pressures.id AS bp_id,
+      blood_pressures.facility_id AS facility_id,
+      cast(EXTRACT(DOY FROM blood_pressures.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS day,
+      cast(EXTRACT(MONTH FROM blood_pressures.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS month,
+      cast(EXTRACT(QUARTER FROM blood_pressures.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS quarter,
+      cast(EXTRACT(YEAR FROM blood_pressures.recorded_at at time zone 'utc' at time zone (SELECT current_setting('TIMEZONE'))) as text) AS year
+  FROM blood_pressures
+  INNER JOIN medical_histories ON (blood_pressures.patient_id = medical_histories.patient_id AND medical_histories.hypertension = 'yes')
```

Comment:
@prabhanshuguptagit I was wondering if we can achieve this by simply merging patients to this view and scoping via `hypertension_only` in the application code and not bother with merging MH + join filter?

Reply:
We can't do that since the `count` aggregation happens in the view. The only way to make it happen is if we do the count also in AR.

Reply:
Ugh, yeah, missed the second part of this view.